### PR TITLE
Fix: Redesign Email Confirmed Page(#1157)

### LINKED ISSUE #1157: Change the email confirmation template according to current UI.
## Current Behaviour
email
The Email confirmation template is according to old UI.

## Expected Behaviour
The email confirmation template should be according to the current UI.

### PR DESCRIPTION
**Pull request by Ram81**
Fixes #1157

### CODE REVIEW

File: apps/accounts/templates/account/email_confirm.html [hunk 1]
```diff
@@ -6,35 +6,54 @@
...
-      <div class="col s12 m4 offset-m4 text-highlight center w-300">
-        <a href="https://evalai.cloudcv.org/auth/login" class="waves-effect waves-dark btn ev-btn-dark blue-grey darken-4">Log In</a>
-      </div>
+    <!-- rendering auth children here -->
+    <div class="auth-cont">
+        <div class="row">
+            <div class="col s12 m6 offset-m3">
+                <div class="left">
+                    <h5><strong><a ui-sref="home" class="dark-link"><img class="auth-logo" src="{% static 'accounts/images/evalai-logo.png' %}"></a></strong></h5>
+                    <p class="text-med-gray auth-home">Evaluating state of the art in AI!</p>
```
Comment (deshraj) [inline apps/accounts/templates/account/email_confirm.html]: Can we remove the `!` at the end?

Comment (deshraj) [reply apps/accounts/templates/account/email_confirm.html]: Rest all looks good. 👍

Comment (Ram81) [reply apps/accounts/templates/account/email_confirm.html]: yeah sure

**Comment by Ram81** [comment]
@deshraj done :+1:

File: apps/accounts/templates/account/email_confirm.html [hunk 2]
```diff
@@ -6,35 +6,54 @@
...
-    <div class="row">
-      <div class="col s12 m4 offset-m4 text-highlight center w-300">
-        <a href="https://evalai.cloudcv.org/auth/login" class="waves-effect waves-dark btn ev-btn-dark blue-grey darken-4">Log In</a>
-      </div>
+    <!-- rendering auth children here -->
+    <div class="auth-cont">
+        <div class="row">
+            <div class="col s12 m6 offset-m3">
+                <div class="left">
+                    <h5><strong><a ui-sref="home" class="dark-link"><img class="auth-logo" src="{% static 'accounts/images/evalai-logo.png' %}"></a></strong></h5>
```
Comment (RishabhJain2018) [inline apps/accounts/templates/account/email_confirm.html]: @Ram81 Can we please align logo and below text with the text `Your email has been verified successfully.`
@deshraj What do you say /

**Comment by RishabhJain2018** [comment]
@Ram81 Also please update the comment using https://help.github.com/articles/closing-issues-using-keywords/

File: apps/accounts/templates/account/email_confirm.html (hunk 2 above)
Comment (Ram81) [reply apps/accounts/templates/account/email_confirm.html]: Sure @RishabhJain2018

**Comment by Ram81** [comment]
@deshraj @RishabhJain2018 please review

**Comment by RishabhJain2018** [comment]
@Ram81 Can you please paste the screenshot here.

**Comment by Ram81** [comment]
abc

**Comment by RishabhJain2018** [comment]
@Ram81 I think you didn't get my point . I was talking of vertical center alignment of EvalAI logo and below text with `Your email has been verified successfully. ` Please let me know if there is still some doubt.

**Comment by Ram81** [comment]
abc
is this good? or you want logo to be left aligned?

**Comment by RishabhJain2018** [comment]
@Ram81 It looks good to me. @deshraj What do you say ??

**Comment by deshraj** [comment]
Looks good to me as well. Thanks @Ram81 :)

**Pull request merged**